static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -399,7 +403,11 @@ define([
         mediator.on('page:media:moreinloaded', initPlayButtons);
 
         initFacia();
-        initMoreInSection();
+
+        if (config.page.section) {
```

Comment:
There are already logic checks inside `initMoreInSection()` function, so I think this extra check would make more sense to do there?